pod/perlhack.pod

```diff
@@ -1166,7 +1166,7 @@ source, and we'll do that later on.
 Gisle Aas's "illustrated perlguts", also known as I<illguts>, has very
 helpful pictures:
 
-L<https://search.cpan.org/dist/illguts/>
+L<https://metacpan.org/release/RURBAN/illguts-0.49>
```

Comment:
Linking to a specific version is probably not the most helpful solution here.

Reply:
It's kind of the only option because illguts cannot be indexed.

Reply:
The bigger problem is illguts is not visible on metacpan. I believe there is a better option but I don't recall it currently.

Reply:
The contents are visible via st.aticpan.org: https://st.aticpan.org/source/RURBAN/illguts-0.49/index.html

Reply:
If we have no better option than mentioning the version number in the hyperlink, then that's preferable IMO to leaving the search.cpan.org link in.

Reply:
Fair. We may just have to keep updating that link then though.

Reply:
To be clear, I was suggesting to use the st.aticpan.org URL I linked here, because the metacpan link does not display the contents as search.cpan.org used to do here years ago (so was the original intention).

Reply:
It looks like this change request was missed.